sync/src/synchronizer/block_process.rs

```diff
@@ -9,7 +9,7 @@ use ckb_types::{packed, prelude::*};
 pub struct BlockProcess<'a> {
     message: packed::SendBlockReader<'a>,
     synchronizer: &'a Synchronizer,
-    peer: PeerIndex,
+    _peer: PeerIndex,
```

Comment:
Why?

Reply:
- https://github.com/nervosnetwork/ckb/pull/2081/files/e99b150daee3b942b6d36753a8a59fb2a7620cae#diff-402ee94ba649e0f53da25c4da84ec58dL41

- https://github.com/nervosnetwork/ckb/pull/2081/files/e99b150daee3b942b6d36753a8a59fb2a7620cae#diff-402ee94ba649e0f53da25c4da84ec58dL56

The above two code has changed/removed, `peer` is unnecessary for `BlockProcess`.
However, removing `peer` from `BlockProcess` would affect many test codes. I just change to the field name to `_peer` to disable the compiler noise.